### ISSUE TITLE
How to completely replace/overhaul election process?

### ISSUE THREAD

**Issue by karilen** [holacracyone/Holacracy-Constitution#221]
The current election process has some pieces, like the objection round, and also issues like now knowing who is interested in the role aside from the nomination round, that make it clunky and personal, especially when people don't understand the roles or are concerned about subordinates not doing a good job in the elected role. I wonder about stating how the first time elections should be done, it should be done in a certain way to allow for more learning and explanation or something. Also I wonder about Lead Link not doing Secretary, but it is a great way for them to add value often. Also when there is an objection, coaches try to both break power dynamics and avoid weird personal attacks, so the objection part is weird, which is back to why it needs some work.

**Comment by brianjrobertson** [comment]
Anyone have ideas for possible improvements to test?

**Comment by karilen** [comment]
Usually I test objections if I see the power dynamics, but if it is too personal I take advantage of throwing out the nomination to avoid the personal attacks from the objector, if that helps. I think a whole new election app is needed.

**Comment by tylerdanke** [comment]
Today I experienced one example of a tension with the current process for elections process: 

There were 5 people in a meeting. Two of them voted for candidate A. 1 each voted for candidates B, C, and D. 3 out of 5 people were not in favor of Candidate A. 

Yes they could object but it better reflect the will of the circle if those 3 people were able to, or automatically able to reallocate their vote. They can change their vote now but all 3 of those people don't know if candidate B, C, or D would be more popular with the rest of them. Ranked voting would allow each voter's vote to count in a second round. So if the voter that voted for Candidate B voted for BCA in that rank, then when candidate B didn't get 51% in the first round, then Candidate C got their vote. 

Maybe a more simple way to process this is that there needs to be at least 51% of the votes in order for it to proceed to a proposal by the Facilitator. 

Systems like OPA Vote make this ranked voting very simple with as many candidates as you want.

**Comment by LouisChiquet** [comment]
I feel like the process you propose @tylerdanke is for “rescuing” those three people because they didn't “dared” to object. I believe it'd more be about coaching for this instance, and also maybe they simply knew that they can ask a reelection at any time, so “yeah, we may try with Candidate A even if I'm not in favour of him”, happens often to me, but given the agility, there's no issues with like trying and asking even an exceptional governance meeting if you want for reelection. Plus there's a “majority thing” which isn't compatible in what I think of Holacracy in my opinion and would put pressure on people for accepting the proposal as 51% of the people voted in favour of this specific candidate plus add complexity for some circle which can sometimes have 30 members attending and it can be longer.

For coming back to @karilen's post, what do you mean by “_Also I wonder about Lead Link not doing Secretary, but it is a great way for them to add value often._”, you'd suggest not having the possibility of the Lead Link (now Circle Lead) being able to have the Secretary role in “his/her” Circle? I'd personally go against this, as the Secretary role is generally considered well... For the former Secretary which was the assistant of the former manager which is now Circle Lead. And funnily enough, often happens for the Circle Lead to ends up Secretary, which from a symbolic point of view with just the name, breaks the former “tradition” and helps in the shift.

I perfectly agree on the power dynamic thing. And what's interesting is that we use the Objection round for deciding, de facto the Objection criteria, but it's rather tough to use criteria 4 for instance, which is totally not suited. So maybe thinking about specific criteria? Or when an objection touches “a Partner personally” then it's valid and you need to pass like you said? However would mean the Constitution enters into more details and “Human”. Easier would be to have no defined process and let each org. define their own, but wouldn't help for the goal Holacracy intends in the end. So I say “Sure !”, but not sure what to propose for addressing this.

**Comment by sitron** [comment]
It's true that election process often feel personal. But i guess this is what election is... nominating someone!
I have no idea how to improve it, but i would definitely try _not_ to make it more complicated: in my experience it often feels like over-engineered and time consuming in regards to the output

**Comment by MiekeByerley** [comment]
Hmm, interesting that this popped up. I forwarded a proposal (back in 2015) that stemmed from similar observation, but from a different angle (I identified not 2 Role types but 3). Aside from having the 2 conventional methods of filling a role **Assignment** and **Election**, I propossed a 3rd which was a Hybrid of the two and I coined it "Appointment". I'll attach the original proposal here for anyone interested (I don't know whether it was ever tabled for consideration or not)
[HOLACRACY CONSTITUTION v 5 proposal.docx](https://github.com/holacracyone/Holacracy-Constitution/files/3033067/HOLACRACY.CONSTITUTION.v.5.proposal.docx)

**Comment by LouisChiquet** [comment]
Thanks for sharing this @MiekeByerley. It's a deep proposal, and haven't been sure to understand all. What would help me to get it, is if you could share the reasons that pushed you to “Core Non-defined Roles” definition or saying it exist? Knowing that you can't add anything to the Circle Lead role, and was wondering your reference about “Context-Link”? Or “Cross-Link” which is something that no longer exist in 5.0 (or it does, but under a different shape)? As explications are a bit light on the document for me to understand properly.
Thanks in advance for the clarifications, that'd be useful.

**Comment by St-Ex** [comment]
On my journey to Holacracy, I stumbled on this tread. I'd like to share a bit of information regarding election. 

First of all, there is no perfect process when you're using ordering (ie: making an ordered list of candidate, even a list of one). It's been proven by [Arrow's impossibility theorem](https://en.wikipedia.org/wiki/Arrow%27s_impossibility_theorem)). Also, ordering people is a trigger for egos as candidates receive a feedback as *better* or *worse* than others.

There is an interesting system, it's called Majority Judgment by Rida Laraki et Michel Balinski. [Here](https://www.youtube.com/watch?v=ZoGH7d51bvc&feature=emb_title) a nice video explaining how it could be apply to french election (at 15:00, English subtitles available).
In short, you are not ordering candidates, grading them independently regarding their fit for the role. Designated candidate is the one with the best median grade. This also offer a very good side effect : only bad grades → no-one is fit for it. you might then carry this tension to super-circle.

As @sitron mention, it might be over-engineered and should be used for critical role (or non-stable circle).

**Comment by brianjrobertson** [comment]
While I still really want an alternative to the current election process, it's not going to make it into v5.0 before release, so I'm closing this issue for now.  I'll open a new issue whenever I have concrete suggestions to offer for potential inclusion in v5.1+, and I invite others to do the same.
